exercises/circular-buffer/circular_buffer_test.go

```diff
@@ -19,11 +19,11 @@ import (
 	"testing"
 )
 
-const testVersion = 2
+const targetTestVersion = 3
 
-func TestTestVersion(t *testing.T) {
-	if TestVersion != testVersion {
-		t.Errorf("Found TestVersion = %v, want %v.", TestVersion, testVersion)
+func TesttestVersion(t *testing.T) {
```

Comment:
it seems to make sense to keep this as `testTestVersion` but I don't feel strongly either way